guide/src/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css' />
+        <link href='https://fonts.googleapis.com/css?family=Roboto:300,400' rel='stylesheet' type='text/css' />
```

Comment:
Within Troposphere, I think the InstanceLaunchWizard occurrences of `<RaisedButton />` have `font-weight: 500`, but I can confirm that.